From 55d403100d30e361132790eae9b5c951d2db4843 Mon Sep 17 00:00:00 2001
From: Roshni Shah <roshnishah318@gmail.com>
Date: Fri, 22 May 2020 19:11:00 -0400
Subject: [PATCH] Create Patch File

---
 client3.py | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/client3.py b/client3.py
index 5843fb1..eafac90 100644
--- a/client3.py
+++ b/client3.py
@@ -42,7 +42,7 @@ def getRatio(price_a, price_b):
 	""" Get ratio of price_a and price_b """
 	""" ------------- Update this function ------------- """
 	""" Also create some unit tests for this function in client_test.py """
-	if(price_b == 0)
+	if(price_b == 0):
 		return
 	return price_a/price_b
 
@@ -56,8 +56,10 @@ if __name__ == "__main__":
 		quotes = json.loads(urllib.request.urlopen(QUERY.format(random.random())).read())
 
 		""" ----------- Update to get the ratio --------------- """
+		prices = {}
 		for quote in quotes:
 			stock, bid_price, ask_price, price = getDataPoint(quote)
+			prices[stock] = price
 			print ("Quoted %s at (bid:%s, ask:%s, price:%s)" % (stock, bid_price, ask_price, price))
 
-		print ("Ratio %s" % getRatio(price, price))
+		print ("Ratio %s" % getRatio(prices['ABC'], prices['DEF']))
-- 
2.21.0 (Apple Git-122.2)

